### DISCUSSION TITLE
How to add private NPM package token to install command?

### DISCUSSION THREAD

**Discussion by rashajkbx** [vercel/next.js#48399] [Help]
### Summary

I have a font awesome pro license that requires our token to install the font awesome pro package.
I have fixed up our workflows by adding `npm config set '//npm.fontawesome.com/:_authToken' SECRET_TOKEN` before `npm install`.

On Vercel deploy, are we able to use private variables? I do not want this token exposed in the build logs.

**Comment by wwwdepot** [comment]
https://vercel.com/guides/using-private-dependencies-with-vercel
